.github/workflows/release.yml

```diff
@@ -28,11 +28,11 @@ jobs:
       - name: Run Tests
         run: npm test --if-present
       - name: Release
-        uses: cycjimmy/semantic-release-action@v2
+        uses: cycjimmy/semantic-release-action@v3
         with:
-          dry_run: false
```

Comment:
This should be `false`. In eslint it's `true` because we are not yet ready for initial release

Reply:
Amended, Thank you